UI/home.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="stylesheet" href="css/home2.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+  </head>
+  <body>
+    <header>
+      <div class="navbar">
+        <a href="#home" class="active"><i class="fa fa-home"></i> Home</a>
+        <a href="#add_item"><text> <i class="fa fa-fort-awesome-alt"> Fast-Food-Fast - Admin Panel</i></text></a>
```

Comment:
I do see a tag 'text', please confirm if it's meant to be here. Otherwise if that gets sort, then you can merge